packages/report-server/src/aggregator/Aggregator.ts

```diff
@@ -14,13 +14,22 @@ type PeriodParams = {
 };
 
 export class Aggregator extends BaseAggregator {
+  aggregationToAggregationConfig = (aggregation: string ) => ({
+    type: aggregation
+  });
+
   async fetchAnalytics(
     dataElementCodes: string[],
+    aggregationList: string[] | undefined,
```

Comment:
Tbh I suspect I will refactor this soon as we will need to be able to pass in aggregation config in the config itself, but that can be done when it's needed 👍

Reply:
Yeah I agree, I was initially allowing object here for that. But I don't want to push too much if it's not needed now (I don't think those STRIVE vizes for solutions will need it anyway)